ServerCore/Pages/Puzzles/Feedback.cshtml.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using ServerCore.DataModel;
+using ServerCore.ModelBases;
+
+namespace ServerCore.Pages.Puzzles
+{
+    /// <summary>
+    /// Model for the author/admin's view of the feedback items for a specific puzzle
+    /// /used for viewing and aggregating feedback for a specific puzzle
+    /// </summary>
+    public class FeedbackModel : EventSpecificPageModel
+    {
+        private readonly ServerCore.Models.PuzzleServerContext _context;
```

Comment:
PuzzleServerContext just moved to the ServerCore.DataModel namespace, so you'll need to remove namespace qualification where you're using it